tests/test_grid.py

```diff
@@ -6,10 +6,19 @@
 from mesa.space import Grid, SingleGrid, MultiGrid
 
 # Initial agent positions for testing
-#              X ---- >
-TEST_GRID = [[0, 1, 0, 1, 0],   # Y
-             [0, 1, 1, 0, 0],   # |
-             [0, 0, 0, 1, 0]]   # V
+#
+# --- visual aid ----
+#   0 0 0
+#   1 1 0
+#   0 1 0
```

Comment:
visual-aid part in the comment is the grid to be perceived visually. Imagine it to be the real world grid where-in x direction has 3 units and y direction has 5 units.The x direction goes from left to right while the y direction goes from bottom to top.  On the other hand TEST_GRID part (which is out of comment scope) is the actual representation in python list of list so as to get the indexing equivalent (one-to-one mapping)  of real word grid..